airflow/models/renderedtifields.py

```diff
@@ -26,7 +27,7 @@
 from airflow.models.base import ID_LEN, Base
 from airflow.models.taskinstance import TaskInstance
 from airflow.serialization.helpers import serialize_template_field
-from airflow.settings import IS_K8S_OR_K8SCELERY_EXECUTOR, json
```

Comment:
We should remove this variable

Reply:
@kaxil I think it's used elsewhere

Reply:
oh yea it still needs to be used in `airflow/www/extensions/init_jinja_globals.py` and `airflow/www/views.py` 

```
airflow/models/renderedtifields.py:from airflow.settings import IS_K8S_OR_K8SCELERY_EXECUTOR, json
airflow/models/renderedtifields.py:        if IS_K8S_OR_K8SCELERY_EXECUTOR:
airflow/settings.py:IS_K8S_OR_K8SCELERY_EXECUTOR = conf.get('core', 'EXECUTOR') in {
airflow/www/extensions/init_jinja_globals.py:from airflow.settings import IS_K8S_OR_K8SCELERY_EXECUTOR, STATE_COLORS
airflow/www/extensions/init_jinja_globals.py:            'k8s_or_k8scelery_executor': IS_K8S_OR_K8SCELERY_EXECUTOR,
airflow/www/views.py:        if not settings.IS_K8S_OR_K8SCELERY_EXECUTOR:
tests/models/test_renderedtifields.py:    @mock.patch("airflow.models.renderedtifields.IS_K8S_OR_K8SCELERY_EXECUTOR", new=True)
tests/models/test_taskinstance.py:    @patch("airflow.models.renderedtifields.IS_K8S_OR_K8SCELERY_EXECUTOR", new=True)
tests/www/test_views.py:        with mock.patch.object(settings, "IS_K8S_OR_K8SCELERY_EXECUTOR", True):
```